pkg/service/keystore/service.go

```diff
@@ -126,6 +126,17 @@ func (s Service) GetKey(ctx context.Context, request GetKeyRequest) (*GetKeyResp
 	}, nil
 }
 
+func (s Service) DeleteKey(ctx context.Context, request DeleteKeyRequest) (*DeleteKeyResponse, error) {
```

Comment:
since there's no response I think this should just return `error`

Reply:
Sure thing. I think I was trying to be consistent with some other stuff I've seen somewhere in the ssi-server, but throwing just an error is fine too .